sym_object.py

```diff
@@ -56,71 +80,144 @@ def __init__(self, x, y, width, height, scene, component_name, name,
         self.y = self.scenePos().y()
         self.state.current_sym_object = self
 
-    # Create an instantiated simobject for the symobject
+    """Get additional info on params such as default values  after
```

Comment:
docstrings go after the definition. See PEP8 (https://www.python.org/dev/peps/pep-0008/#documentation-strings)